UnitySDK/Assets/ML-Agents/Scripts/LearningBrain.cs

```diff
@@ -126,7 +125,8 @@ public IEnumerable<string> GetModelFailedChecks()
         /// <inheritdoc />
         protected override void DecideAction()
         {
-            if (m_IsControlled)
+            m_Batcher?.SendBrainInfo(name, m_AgentInfos);
+            if (m_Batcher != null)
```

Comment:
does a check for null need to be here?  What's the case when DecideAction is called when the batcher is not yet set?

Reply:
If there is no batcher, I suppose the Inference behavior is used